cfgov/cfgov/urls.py

```diff
@@ -299,8 +298,9 @@ def empty_200_response(request, *args, **kwargs):
     ),
 
     re_path(
-        r'^data-research/consumer-complaints/$',
-        ComplaintLandingView.as_view(),
+        r'^data-research/consumer-complaints/$', TemplateView.as_view(
```

Comment:
Don't you still need to keep CLV around to leverage all of the various [contextual data](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L28-L39)? It's already configured to use [`complaint/complaint-landing.html`](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L26) so it should already use your new template.

Reply:
Hmmmmm… you're right. Updated in https://github.com/cfpb/consumerfinance.gov/pull/6357/commits/bf6a3d50a44abc510459ffd2dacdb5fd06c7f94d

The template I migrated has three variables it uses to show notifications: `data_down`, `narratives_down`, and `technical_issues`. Do you see anywhere where the `ccdb_content_updates` [flag](https://github.com/cfpb/consumerfinance.gov/blob/d61886b2316781937248d4483f1cb002c7cf62d7/cfgov/legacy/views/complaint.py#L36) is used? Can we delete that?

Reply:
PR'd the flag removal [here](https://github.com/cfpb/consumerfinance.gov/pull/6359)